### PR TITLE
Add first impression for draw GUI

### LINKED ISSUE #1: GUI mock

### PR DESCRIPTION
**Pull request by JBenda**
closes #1

### CODE REVIEW

File: Notes/Design/GUI.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+	* redo
+		* new canvas
+		* redo step
+		* back end of painting state
+* painting state:
+	* color dropper
+	* color racks
+	* go to modeling state
+* modeling state
+	* rack
```
Comment (t-erxleben) [inline Notes/Design/GUI.md]: pls change word to rake

File: Notes/Design/GUI.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+* you can have either painting or modeling tools <- (New image -> painting state -> modeling state -> finish)
+* movement direction and intensity it done with mouse input in normal mode
+
+* every state
+	* color picker
+		* different colors (<- select to change?)
+	* redo
+		* new canvas
+		* redo step
+		* back end of painting state
```
Comment (t-erxleben) [inline Notes/Design/GUI.md]: What do you mean by back end of painting state?

**Review by t-erxleben — CHANGES_REQUESTED** [review]
See comments in GUI.md

File: Notes/Design/GUI.md (hunk 2 above)
Comment (JBenda) [reply Notes/Design/GUI.md]: _Back to end of painting state_: the canvas state, before the first rake will be applied

File: Notes/Design/GUI.md (hunk 1 above)
Comment (JBenda) [reply Notes/Design/GUI.md]: sure :sweat_smile:

File: Notes/Design/GUI.md (hunk 2 above)
Comment (t-erxleben) [reply Notes/Design/GUI.md]: good idea; did not thought of this one

**Review by t-erxleben — APPROVED** [review]

**Pull request merged**